samples/modules/lvgl/multi_display/sample.yaml

```diff
@@ -0,0 +1,22 @@
+sample:
+  description: LVGL multi-display sample
+  name: LVGL Multi-display
+common:
+  modules:
+    - lvgl
+  harness: console
+  filter: dt_comp_enabled("zephyr,displays")
+  tags:
+    - samples
+    - display
+    - lvgl
+    - gui
+  harness_config:
+    fixture: fixture_display
+    type: one_line
+    regex:
+      - "\\[\\w+ free bytes, \\w+ allocated bytes, overhead = \\w+ bytes | lvgl in malloc mode\\]"
```

Comment:
Kinda divided on the whole sample with console harness. Had a larger discussion about this not too long ago. Personally I'd avoid adding it since things like the allocation printout is implementation detail inside of `sys_heap`. If someone was to change the text for whatever reason, he'd need to adjust the LVGL sample. That just feels odd.